pac4j-jwt/src/test/java/org/pac4j/jwt/JwtTests.java

```diff
@@ -62,6 +72,14 @@ public void testGenerateAuthenticate() {
         assertEquals(profile.getVerified(), fbProfile.getVerified());
     }
 
+    private FacebookProfile createProfile() {
+        final FacebookProfile profile = new FacebookProfile();
```

Comment:
The `JwtAuthenticator` implements `TokenAuthenticator` which expects at least a `HttpProfile`, that's why it doesn't work with `CommonProfile` (which is an ancestor).